hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/TestKeyValueContainer.java

```diff
@@ -216,8 +213,6 @@ public void testEmptyContainerImportExport() throws Exception {
 
   @Test
   public void testContainerImportExport() throws Exception {
-    assumeFalse(schemaVersion.equals(OzoneConsts.SCHEMA_V3));
```

Comment:
Could you please add one more test of exporting same container twice.

Reply:
I think we already got a stricter case in this file `public void concurrentExport()`.

Reply:
OK.